README.md

```diff
@@ -99,8 +100,8 @@ parameters, in which case they do not specify `oauth2` or `zcap` properties.
 
 #### Testing locally
 
-If you want to test your implementations for endpoints running locally, you can
-create a configuration file `.localImplementationsConfig.cjs` in the root
+To test implementations with endpoints running locally, create a configuration file named
+either `.localImplementationsConfig.cjs` or `localImplementationsConfig.cjs` in the root
 directory of the test suite.
```

Comment:
Question: do we need to support two options? Would be potentially less confusing if we can just support the former, but I recognize there may be historical reasons to do both

Reply:
major reason for both is that the implementations library doesn't have semantic versioning because it's not released via npm or github packages so yeah everything must be a minor feature right now.